src/utils/chap.ts

```diff
@@ -52,7 +50,81 @@ export function buildChapPayload(historicalDisease: ClickhouseHistoricalDisease[
 				};
 			})
 		},
+		// n_periods: 24
 	};
 
-  return payload;
+	// console.log(JSON.stringify(payload))
+
+  return (payload);
 }
+
+function fixPayload(payload: any) {
+	let fixedPayload = JSON.parse(JSON.stringify(payload)); // Deep copy to avoid mutation
+    const requiredMonths = 24; // At least 2 years of data
+    const currentYearMonth = new Date().getFullYear() * 100 + new Date().getMonth() + 1; // YYYYMM format
+    const earliestRequiredPeriod = currentYearMonth - requiredMonths;
+
+    // Convert "value" fields from string to number
+    // @ts-ignore
+		fixedPayload.features.forEach(feature => {
+			// @ts-ignore
+        feature.data.forEach(entry => {
+            if (typeof entry.value === "string") {
+                entry.value = Number(entry.value);
+            }
+        });
+    });
+
+    // Remove duplicate orgUnits in orgUnitsGeoJson
+    let seenOrgUnits = new Set();
+		// @ts-ignore
+    fixedPayload.orgUnitsGeoJson.features = fixedPayload.orgUnitsGeoJson.features.filter(feature => {
+        if (seenOrgUnits.has(feature.id)) {
+            return false; // Remove duplicate
+        }
+        seenOrgUnits.add(feature.id);
+        return true;
+    });
```

Comment:
The payload generated has duplicate orgs and this is going to make the predictions bug out.